### ISSUE TITLE
README/KDE STORE UPDATE NEEDED

### ISSUE THREAD

**Issue by p134c0d3** [samirgaire10/com.samirgaire10.brave#1]
Just found the plasmoid, and it has me intrigued. Though, you may not get very many people to install it because neither the KDE Store page or the GitHub README tell the user exactly what this plasmoid is supposed to do. Is it just the browser itself implemented as a plasmoid? Or is it a feature of the browser? As of right now, I have no clue what this is supposed to do, so you may want to update both the KDE Store listing and the GH readme file indicating the purpose and functionality of the plasmoid. 
Hopefully this is seen in the air of helpfuflness rather than malice!

**Comment by samirgaire10** [comment]
A simple plasmoid to allow quick access to the brave search site directly from the KDE panels/desktop.
So basically I use  [pling ](https://www.pling.com/p/1977771).      or     [GitHub source ]( https://github.com/dark-eye/com.darkeye.chatGPT).
source code and change it in a way for my use case or my need ... brave, google-gemini, deepl translator because it usefull for me so decide to publish it ... I did not want to do multiple steps to do one thing so I made plasmoid ... I can access things I needed just with one click

Yes ... I will try to update the readme file  -------------------------------------------------------------------  later
yes --- it was helpful 
wow this was my first isseus in github i was びっくりした！
thank you
